modin/experimental/xgboost/test/test_dmatrix.py

```diff
@@ -0,0 +1,166 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+import pytest
+
+import modin.experimental.xgboost as mxgb
+import modin.pandas as pd
+import numpy as np
+
+import multiprocessing as mp
+import ray
+
+import xgboost as xgb
+
+from sklearn.metrics import accuracy_score
+
+
+ray.init(log_to_driver=False)
+
+num_cpus = mp.cpu_count()
+
+
+rng = np.random.RandomState(1994)
+
+
+def eval_general(data, modin_data, **kwargs):
```

Comment:
Possible, need to rename this function, because this is intended to check equality of dmatrix only, or throwed exceptions.